websites/E/EternalNetwork TM/presence.ts

```diff
@@ -158,17 +158,90 @@ presence.on("UpdateData", async () => {
 			if (pathname.startsWith("/")) presenceData.state = ttl;
 
 			if (pathname.includes("/admin")) {
-				presenceData.state = "Adding new incident 😥 !";
+				presenceData.state = "Adding new incident";
 				presenceData.smallImageText = "Admin Panel";
 				delete presenceData.buttons;
 			}
 
 			if (search.includes("?do=settings"))
-				presenceData.state = "Adding new service !";
+				presenceData.state = "Adding new service";
 			delete presenceData.buttons;
 
 			break;
 		}
+		case `et-log.${etrnl}`:
+		case `www.et-log.${etrnl}`: {
+			presenceData.smallImageKey = "eternallogistic_logo";
```

Comment:
Optional but we prefer the use of imgur links for future maintainability? 
You could use an enum with those links like [here](https://github.com/PreMiD/Presences/blob/main/websites/L/LiveLinkBio/presence.ts#L15)

Reply:
Thanks for the suggestion @DarkVillager, but I think using Discord Rich Presence Assets is safe enough because I (or my colleagues) can change a specific image without opening a PR here to update the URL of that specific image.

Also, let me know if there are any other necessary changes I need to make.

Reply:
> Thanks for the suggestion @DarkVillager, but I think using Discord Rich Presence Assets is safe enough because I (or my colleagues) can change a specific image without opening a PR here to update the URL of that specific image.
> 
> Also, let me know if there are any other necessary changes I need to make.

It’s more so other devs can edit & update things if this ever gets abandoned. But you indeed don’t have to.